runtime/src/iree/hal/drivers/vulkan/dynamic_symbol_tables.h

```diff
@@ -319,7 +319,7 @@ namespace vulkan {
   INS_PFN(EXCLUDED, vkGetDisplayPlaneCapabilitiesKHR)                   \
   INS_PFN(EXCLUDED, vkGetDisplayPlaneSupportedDisplaysKHR)              \
   INS_PFN(OPTIONAL, vkGetPhysicalDeviceCalibrateableTimeDomainsEXT)     \
-  INS_PFN(OPTIONAL, vkGetPhysicalDeviceCooperativeMatrixPropertiesKHR)  \
+  INS_PFN(EXCLUDED, vkGetPhysicalDeviceCooperativeMatrixPropertiesNV)   \
```

Comment:
Were these Vulkan changes intended to be included here? They seem unrelated - some git accident?

Reply:
Rebase failure, sorry :(